### DISCUSSION TITLE
Minimum required version of unixODBC for nanodbc?

### DISCUSSION THREAD

**Discussion by beemike5** [nanodbc/nanodbc#344] [Q&A]
It might have been mentioned somewhere but I could not find it:
What is the minimum version number requirement of **unixODBC** for **nanodbc**?

Thanks for your answers in advance!

**Comment by mloskot** [comment]
`¯\_(ツ)_/¯`

The only trace of unixODBC version is this 
https://github.com/nanodbc/nanodbc/blob/62ae5cbb93ede814bc569787f5dde368d4ad4582/test/sqlite_test.cpp#L45-L48

Get the latest and the greatest, I'd say.
